### PR TITLE
Remove string addition

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
@janisozaur Is there a way to prevent `STR_SOMETHING + 1` being possible?

File: src/openrct2/windows/staff.c [hunk 1]
```diff
@@ -290,7 +290,7 @@ uint32 window_staff_page_enabled_widgets[] = {
 	(1 << WIDX_TAB_3)
 };
 
-static const uint32 staffCostumeNames[] = {
+const rct_string_id StaffCostumeNames[] = {
```
Comment (Gymnasiast) [inline src/openrct2/windows/staff.c]: I'm not sure this should be defined in a window definition. The peep/staff.c file is a better option IMO.

File: src/openrct2/windows/ride.c [hunk 2]
```diff
@@ -1087,7 +1087,7 @@ static const rct_string_id MusicStyleNames[] = {
 	STR_MUSIC_STYLE_CANDY_STYLE,
 };
 
-static const rct_string_id RideModeNames[] = {
+const rct_string_id RideModeNames[] = {
```
Comment (Gymnasiast) [inline src/openrct2/windows/ride.c]: I'm not sure this should be defined in a window definition.

**Comment by janisozaur** [comment]
@marijnvdwerf no, I don't think it is possible, at least not in the current form. Can you provide an actual example of what you want prevented?

**Comment by IntelOrca** [comment]
@marijnvdwerf not in C, you can in C++ though.

File: src/openrct2/ride/ride_data.c [hunk 3]
```diff
@@ -3141,3 +3141,43 @@ const track_colour_preset_list RideColourPresets[] = {
...
+        STR_RIDE_MODE_INTENSE,
+        STR_RIDE_MODE_BERSERK,
+        STR_RIDE_MODE_HAUNTED_HOUSE,
+        STR_RIDE_MODE_CIRCUS_SHOW,
+        STR_RIDE_MODE_DOWNWARD_LAUNCH,
+        STR_RIDE_MODE_CROOKED_HOUSE,
+        STR_RIDE_MODE_FREEFALL_DROP,
+        STR_RIDE_MODE_CONTINUOUS_CIRCUIT_BLOCK_SECTIONED,
+        STR_RIDE_MODE_POWERED_LAUNCH,
+        STR_RIDE_MODE_POWERED_LAUNCH_BLOCK_SECTIONED_MODE,
```
Comment (Gymnasiast) [inline src/openrct2/ride/ride_data.c]: Shouldn't this end with `RIDE_MODE_COUNT` ?

Comment (IntelOrca) [inline src/openrct2/ride/ride_data.c]: Why?

Comment (Gymnasiast) [inline src/openrct2/ride/ride_data.c]: ... Wait, didn't read properly. Sorry about this.

**Pull request merged**